### ISSUE TITLE
Bug: Democracy screen not working

### ISSUE THREAD

**Issue by silva-fj** [litentry/litentry-app#379]
Error logs: 

```
 ERROR  The action 'NAVIGATE' with payload {"name":"Democracy"} was not handled by any navigator.

Do you have a screen named 'Democracy'?

If you're trying to navigate to a screen in a nested navigator, see https://reactnavigation.org/docs/nesting-navigators#navigating-to-a-screen-in-a-nested-navigator.
```

**Comment by silva-fj** [comment]
The same issue for My identity screen:

```
The action 'NAVIGATE' with payload {"name":"My Identity","params":{"address":"14mwSGdhdrAA3pGoKSX1tWguFREswWucAsr7kcHbdsf7fU7Q"}} was not handled by any navigator.

Do you have a screen named 'My Identity'?

**Comment by ahoseinian** [comment]
Sorry i think I broke them moving during the navigation refactoring...

**Comment by ahoseinian** [comment]
fixed in #386
